src/insns/malformed_no_check.adoc

```diff
@@ -0,0 +1,3 @@
+NOTE: This instruction does not perform a <<section_cap_malformed,malformed>>
+bounds check, and so can propagate a <<section_cap_malformed,malformed>>
+tagged capability.
```

Comment:
If we're specifying this for every instruction, is there a reason bounds are special. Why not also say the same about reserved bits and permission bits?

Reply:
good point